src/app/pages/home/home.jsx

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+var StoreMixin = require('fluxible').StoreMixin;
+var GoogleMapsStore = require('../../maps/google-maps-store');
+var NavLink = require('../../nav/nav-link');
+var loadGoogleMaps = require('../../maps/load-google-maps');
+
+/**
+ * Home page.
+ */
+
+module.exports = React.createClass({
+  mixins: [StoreMixin],
+  statics: {
+    storeListeners: [GoogleMapsStore]
+  },
+  getInitialState: function () {
+    var venues = this.props.context.getStore('venue').getVenues();
```

Comment:
React points this out [as an antipattern](http://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html), because it means this state is duplicated here. If the venues list changes, it won't change on the home page.